File: CVR/03-07-23/Python/reverse_array.py


arr = [23,4,5,2,6,-90,78,9,98]

a = 0 
b = len(arr)-1 

while a<b:
    temp = arr[a]
    arr[a] = arr[b]
    arr[b] = temp
    a+=1
    b-=1

print(arr)

File: stanley/b26/02-12-2024/ReverseArray.java
import java.util.Arrays;

class ReverseArray{
    public static void main(String[] args){
        int[] arr ={1,4,2,7,5,8,3};
        int i = 0, j = arr.length-1;
        while(i < j){
            int temp = arr[i];
            arr[i] = arr[j];
            arr[j] = temp;
            i+=1;
            j-=1;
        }
        // for(int k = 0; k < arr.length; k++){
        //     System.out.print(arr[k]+" ");
        // }
        // for(int e : arr){
        //     System.out.print(e+" ");
        // }
        System.out.println(Arrays.toString(arr));
    }
}